packages/prisma-fabbrica/src/generatorNext.ts

```diff
@@ -0,0 +1,11 @@
+import { generatorHandler } from "@prisma/generator-helper";
+import { onGenerate } from "./onGenerate";
+
+generatorHandler({
+  onManifest: () => ({
+    prettyName: "fabbrica",
+    defaultOutput: "../src/__generated__/fabbrica",
+    requiresGenerators: ["prisma-client"],
```

Comment:
Difference between `generator.ts` and `generatorNext.ts` seems only this `requiresGenerators`'s value, doesn't it?
And I understood the 2 generator files will force users to select fabbrica provider name, `prisma-fabbriaca` or `prisma-fabbrica-next`.

Actually, we don't need the `requiresGenerators` value because fabbrica's generator procedure don't use the artifact of `prisma-client-js` (nor `prisma-client`). 

So I think we can support both `prisma-client-js` and `prisma-client` without adding new generator bin. All we need is remove the `requiresGenerators`.

How about this ?

Reply:
> Actually, we don't need the requiresGenerators value because fabbrica's generator procedure don't use the artifact of prisma-client-js (nor prisma-client).
> So I think we can support both prisma-client-js and prisma-client without adding new generator bin. All we need is remove the requiresGenerators.
> How about this ?

I totally agree with this! I’ve reverted my previous commit, and now the only change is the removal of requiresGenerators.